harness/tests/common/test_schemas.py

```diff
@@ -100,31 +105,46 @@ def run_completeness(self) -> None:
             errors = expconf.completeness_validation_errors(self.case, url)
             if not errors:
                 continue
-            raise ValueError(f"'{self.name}' failed against {url}:\n - " + "\n - ".join(errors))
+            raise ValueError(
+                ("'{}' failed completeness validation against {}:\n - " "\n - ")
+                .format(self.name, url)
+                .join(errors)
+            )
+
+    def run_sanity_errors(self) -> None:
+        if not self.sanity_errors:
+            return
+        self.run_errors(self.sanity_errors)
 
-    def run_errors(self) -> None:
-        if not self.errors:
+    def run_completeness_errors(self) -> None:
+        if not self.completeness_errors:
             return
-        for url, expected in self.errors.items():
+        self.run_errors(self.completeness_errors, test_type="completeness")
+
+    def run_errors(self, error_cases: Any, test_type: Optional[str] = "sanity") -> None:
```

Comment:
don't use default arguments for "internal" helper functions like this.  Make it required so it is explicit what the caller is trying to specify, rather than have half the callers depend on defaults to communicate their intent.

Reply:
possibly also call this `_run_errors` to distinguish it from the "higher" functions that invoke this one.